### PR TITLE
List of cards added in information view

### PR DESCRIPTION
**Pull request by LukasJonsson99**
Co-authored-by: Daniel Karlsson <Dolvur@users.noreply.github.com>

Beginning of the information implementations. Every card has a picture that leads to a new screen which include information about that topic.

### CODE REVIEW

**Comment by Neathan** [comment]
Looks like a good start.

The header seems like its uneven but i guess that would be resolved in the process of further development.

**Comment by Dolvur** [comment]
Remember to localize the titels for the cards before merging

**Review by KaptenSkruvkork — APPROVED** [review]
Looks great! Cant find anything to commnent on

File: src/components/InformationScreen.tsx [hunk 1]
```diff
@@ -1,18 +1,59 @@
...
+  );
+
   return (
-    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-      <Text>Information</Text>
-    </View>
+    <FlatList
+      data={cardsData}
+      renderItem={renderItem}
+      keyExtractor={(item) => JSON.stringify(item)}
```
Comment (EMattfolk) [inline src/components/InformationScreen.tsx]:
```suggestion
      keyExtractor={item => item.title}
```
Could be simplified a bit

File: src/components/info/ActivityInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import { View } from 'react-native';
+import { Text } from 'react-native-paper';
+
+export const ActivityInfo = () => {
+    return (
+      <View>
+        <Text>ActivityInfo</Text>
+      </View>
+    );
+  }
```
Comment (EMattfolk) [inline src/components/info/ActivityInfo.tsx]: All of the new files have CRLF line endings and is missing a newline at the end.

**Review by EMattfolk — APPROVED** [review]

File: src/components/InformationScreen.tsx [hunk 3]
```diff
@@ -1,18 +1,59 @@
...
+import { CalenderInfo } from './info/CalenderInfo';
+import { ValuesInfo } from './info/ValuesInfo';
+import { BehaviouralActivation } from './info/BehaviouralActivation';
 
 const InformationStack = createStackNavigator();
 
-const ViewContent = () => {
+const cardsData = [
+  {
+    title: "BehaviouralActivation",
```
Comment (malin-mallan) [inline src/components/InformationScreen.tsx]: Tänk på att dessa måsta översättas. Nu blir det engelsk rubrik trots svenska som inställning.

Samma gäller såklart med alla rubriker nedan och all text som är inne på dem olika korten

Comment (malin-mallan) [reply src/components/InformationScreen.tsx]: Och borde det inte stå "Behaviour Activation" istället för "BehaviouralActivation"?

**Pull request merged**